src/components/Todo/AddNew.jsx

```diff
@@ -20,16 +21,10 @@ function AddNew( props ) {
 
   return (
     <form className="add-new" onSubmit={handleFormSubmission}>
-      <label htmlFor="myInput">Add new todo</label>
-      <div>
-        <input
-          id="myInput" type="text"
-          value={title}
-          onChange={handleTitleChange}
-          placeholder="Ex: Do cleaning ..."
-        />
-        <button type="submit">Add</button>
-      </div>
+      <InputWithLabel title={title} handleTitleChange={handleTitleChange}>
+        <b>Title</b>
```

Comment:
nice touch in making the font bold. Big however, these days, the  `b` tag is against best practices for some accessibility issues, and styling is recommended instead, something like `font-weight: bold:`